sound/soc/sof/intel/byt.c

```diff
@@ -328,7 +328,9 @@ static int byt_run(struct snd_sof_dev *sdev)
 
 	/* release stall and wait to unstall */
 	snd_sof_dsp_update_bits64(sdev, BYT_DSP_BAR, SHIM_CSR,
-				  SHIM_BYT_CSR_STALL, 0x0);
+				  SHIM_BYT_CSR_XT_SNOOP | SHIM_BYT_CSR_STALL,
+				  SHIM_BYT_CSR_XT_SNOOP);
```

Comment:
does this enable PCIe snooping of the DSP traffic? And without it it's disabled? I'm not an expert in that but in general that seems like a good thing to maintain cache coherency? I don't know why it would be possible to disable it, but maybe my guess is wrong and it's something different entirely.

Reply:
I think it's wrong, will remove this.

Reply:
would that leave snooping disabled? Or does that flag disable snooping when set?